packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -75,19 +93,29 @@ const extractObjectExpression = (node: t.ObjectExpression, meta: Metadata): CSSO
 
       // Time to add this key+value to the CSS string we're building up.
       css += `${kebabCase(key)}: ${value};`;
-    } else if (t.isSpreadElement(prop) && t.isIdentifier(prop.argument)) {
+    } else if (t.isSpreadElement(prop)) {
       // We found a object spread such as: `...mixinIdentifier`.
-      const binding = meta.parentPath.scope.getBinding(prop.argument.name);
-      const resolvedBinding = resolveBindingNode(binding, meta);
+      const expression = tryGettingCalleeFromCallExpression(prop.argument);
+      let resolvedBinding = undefined;
 
-      if (!resolvedBinding) {
-        throw buildCodeFrameError('Variable could not be found', prop.argument, meta.parentPath);
+      if (t.isIdentifier(expression)) {
+        const binding = meta.parentPath.scope.getBinding(expression.name);
+        resolvedBinding = resolveBindingNode(binding, meta);
+
+        if (!resolvedBinding) {
+          throw buildCodeFrameError('Variable could not be found', prop.argument, meta.parentPath);
```

Comment:
did we add this so we could differentiate the errors i guess?

Reply:
Yeah. I can remove this check completely and just ignore the errors when variable could not be found. It won't do anything in that case. What you say?

Reply:
all good let's leave it.